app/src/main/res/values-es/strings.xml

```diff
@@ -68,7 +68,7 @@
     <string name="theme">Tema</string>
     <string name="theme_mode">Modo de tema</string>
     <string name="shapes">FORMAS</string>
-    <string name="thumbnail_roundness">Miniaturas redondeadas</string>
+    <string name="thumbnail_roundness">Redondez de las miniaturas</string>
```

Comment:
I would say Miniaturas redondeadas sounds better.

Reply:
Quiet, there are other translations coming.
Statistics and other option settings.
:)

Reply:
> I would say Miniaturas redondeadas sounds better.

I agree, but I don't think it describes well what it does. We could change it to "Curvatura de las miniaturas", it sounds better.

Reply:
Sure